babel/dates.py

```diff
@@ -862,6 +863,7 @@ def format_timedelta(
     threshold: float = .85,
     add_direction: bool = False,
     format: Literal['narrow', 'short', 'medium', 'long'] = 'long',
+    depth: Literal['shallow', 'full', 'fullest'] = 'shallow',
```

Comment:
I'm not sure these names are the best or most descriptive. No matter what they end up being, they do need to be documented, though.